haskell-src/Concordium/Genesis/Data/Base.hs

```diff
@@ -69,6 +87,34 @@ instance Serialize CoreGenesisParameters where
         genesisFinalizationParameters <- getFinalizationParametersGD3
         return CoreGenesisParameters{..}
 
+-- |Information about the genesis block of the chain. This is not the full
+-- genesis block. It does not include the genesis state. Instead, it is the
+-- minimal information needed by a running consensus.
+--
+-- The intention is that this structured can always be deserialized from a
+-- serialized @GenesisData@ provided the hash of the genesis data is known.
+data GenesisConfiguration = GenesisConfiguration {
+  _gcTag :: !Word8,
```

Comment:
I think it might help to repeat in the comment for the `GenesisConfiguration` type what are the valid values of `_gcTag`.

It's possible to get that information by looking at the serialization code, but it'll help readability to have it in one place.

Reply:
Will add.